api/data_seeding.md

```diff
@@ -14,19 +14,29 @@ We should make requests using the new REST API to archive the top X websites at
 * Automated tests
 
 # Background
-*(terminology and history one needs to know to understand the design)*
+N/A
 
 # High-Level Design
-*(a brief, high-level description of the design; diagrams are helpful)*
+The data_seeder will be a lightweight application that hits the REST API to create new snapshots at given intervals. The list of data to request and the schedule of requests shall be configurable via config file or command line.
 
 # Detailed Design
-*(the design in detail; usually a top-down description)*
+data_seeder is a node.js app running inside a temporary container in our cluster. A very basic starting point is the test branch of CTM/api that simply fires off a bunch of POST requests to the API with a wrapper around it to make it run regularly and with smarter logic.
```

Comment:
Which POST requests are you making? Where are they being sent?  What is the format of the post?